app/src/main/res/values/text_styles.xml

```diff
@@ -34,6 +39,11 @@
         <item name="android:textColor">@color/text_emphasized</item>
     </style>
 
+    <style name="Headline.xLarge">
```

Comment:
```suggestion
    <style name="Headline.XLarge">
```

For the sake of consistency, lets use `XLarge` over `xLarge`, given that we already have `Body.XSmall`